### ISSUE TITLE
Annotation Config doesn't specify the type

### ISSUE THREAD

**Issue by vimorra** [dapr/docs#617]
## Actual Behavior

In the Docs related to the annotation in K8S is not clear which is the type of "dapr.io/config" if it is a string or array? in case of String how we can define multiple configurations?

## Steps to Reproduce the Problem

go at page https://github.com/dapr/docs/tree/master/howto/configure-k8s
This is the current text for Config Annotation: 

dapr.io/config | Tells Dapr which Configuration CRD to use
-- | --

**Comment by yaron2** [comment]
It's the name of the configuration instance.

An app can reference one config object.

**Comment by vimorra** [comment]
Gotcha, but in this case each apllication need to have it own configuration, Could be interesting to enable an array of configurations, based on the fact that some apllications can share part of the configurations

**Comment by vimorra** [comment]
Do you(@yaron2 ) think that can be interesting to add this feature as improvements in DAPR?

**Comment by yaron2** [comment]
Yes, but we need to think through the design and user experience here.

/cc @msfussell
